features/recommendations.feature

```diff
@@ -11,4 +11,24 @@ Background:
 Scenario: The server is running
     When I visit the "Home Page"
     Then I should see "Recommendation RESTful Service" in the title
-    And I should not see "404 Not Found"
\ No newline at end of file
+    And I should not see "404 Not Found"
+
+Scenario: Create a Recommendation
+    When I visit the "Home Page"
```

Comment:
Based on what I learned from Jianwei's PR, I suggest you add a `And I press the "Clear" button` after the "When"